server/apps/aap_mm/aap_mm_datalayer.py

```diff
@@ -54,13 +65,59 @@ def find(self, resource, req, lookup):
         query_keywords = '*:*'
         if 'query' in req['query']['filtered']:
             query_keywords = req['query']['filtered']['query']['query_string']['query']
+            query_keywords = query_keywords.replace('slugline:', 'objectname:')
+            query_keywords = query_keywords.replace('description:', 'captionabstract:')
+
+        fields = {}
+        for filter in req.get('post_filter', {}).get('and', {}):
```

Comment:
I would recommend choosing a different name for the loop variable here, because it shadows the built-in `filter` class.